include/libsemigroups/presentation.hpp

```diff
@@ -530,9 +531,9 @@ namespace libsemigroups {
     //!
     //! \complexity
     //! Linear in the length of the alphabet.
-    void validate_alphabet() const {
+    void throw_if_alphabet_not_duplicate_free() const {
```

Comment:
```suggestion
    void throw_if_alphabet_has_duplicates() const {
```
?

Reply:
This change, if made, would also need to be propagated throughout

Reply:
Much better